templates/endomorphism/batchnormalization.wgsl

```diff
@@ -1,14 +1,47 @@
 
 {%- include "structs.wgsl" -%}
 
+[[block]]
+struct Block {
+    data: [[stride({{ elem_stride }})]] array<{{ elem_type }}>;
```

Comment:
So this could work but many operation on floating-point are only available for f32: https://www.w3.org/TR/WGSL/#float-builtin-functions